### PR TITLE
Feature/project page

### PR DESCRIPTION
**Pull request by andre84o**

### CODE REVIEW

**Comment by Lvhacker1** [comment]
Har du råkat göra ändringar på index.html filen?

**Review by Endorze — CHANGES_REQUESTED** [review]
Your branch is 40 commits behind from the Main branch, this means you're missing 40 file changes of which should be in your Project branch.

**Review by Meazajao — CHANGES_REQUESTED** [review]
You dont have to do the header, Alex is in charge of that :)

**Comment by andre84o** [comment]
Haven't touched master, just copied links from it. header has been removed new updates have been made

**Comment by andre84o** [comment]
> Har du råkat göra ändringar på index.html filen?

Nej hade bara kopierat länkar därifrån

File: websites-css/project.css [hunk 1]
```diff
@@ -2,20 +2,82 @@
...
+    font-family: var(--font-family);
+    text-align: center;
+}
+
+.large {
+    grid-column: 1 / span 2; 
+    }
+
+.footer {
+    padding: 32px;
```
Comment (Endorze) [inline websites-css/project.css]: You should not have this footer class as i have already made a footer, header and its respective css is in the master.css file.

File: websites-css/project.css [hunk 2]
```diff
@@ -2,20 +2,82 @@
...
+    flex-direction: column;
+}
+
+html {
+    height: 100vh;
+}
+
 .header {
     background-color: var(--color-header-bg);
+    margin: 0;
```
Comment (Endorze) [inline websites-css/project.css]: You should not have this header class as i have already made a header, footer and its respective css is in the master.css file.

File: websites-css/project.css [hunk 3]
```diff
@@ -2,20 +2,82 @@
...
+}
+
+
+.box {
+    background-color: var(--color-content-section);
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    border: 1px solid #000;
+    font-size: 1.5rem;
```
Comment (Endorze) [inline websites-css/project.css]: Switch to global variable for scaleability, or use already existing variable --font-size-h3, which also is 1.5rem.

**Review by Endorze — CHANGES_REQUESTED** [review]
You're behind the main branch in commits, you need to make sure you do git pull origin main standing in your branch, and that it actually pulls the code, after this you do git push and if you're getting issues with the git pull command then you can probably find the solution to your issue in the discord groupchat.

File: websites-css/project.css (hunk 1 above)
Comment (andre84o) [reply websites-css/project.css]: i have change it and updated

File: websites-css/project.css (hunk 2 above)
Comment (andre84o) [reply websites-css/project.css]: i have removed it and updated

File: websites-css/project.css (hunk 3 above)
Comment (andre84o) [reply websites-css/project.css]: its change and update

**Review by andre84o — COMMENTED** [review]
fixed

File: websites-css/aboutus.css [hunk 4]
```diff
@@ -15,7 +15,13 @@
     justify-content: center;
     flex-direction: column;
     gap: 70px;
+<<<<<<< HEAD
+    padding: 30px 100px;
+
```
Comment (Endorze) [inline websites-css/aboutus.css]: This is an unresolved merge conflict, you shouldn't push up code where theres merge conflicts that you haven't taken care off.

If you're not sure how to deal with merge conflicts then you should contact Rob

File: websites-css/aboutus.css [hunk 5]
```diff
@@ -29,6 +35,8 @@
     text-align: center;
     border-radius: 10px;
 }
+<<<<<<< HEAD
```
Comment (Endorze) [inline websites-css/aboutus.css]: This is an unresolved merge conflict, you shouldn't push up code where theres merge conflicts that you haven't taken care off.

If you're not sure how to deal with merge conflicts then you should contact Rob

File: websites-html/aboutus.html [hunk 6]
```diff
@@ -40,6 +40,33 @@ <h2 class="page-title">About Us</h2>
             Join us on our journey as we grow into the next generation of skilled frontend developers!</div>
     </div>
 
+<<<<<<< HEAD
```
Comment (Endorze) [inline websites-html/aboutus.html]: This is an unresolved merge conflict, you shouldn't push up code where theres merge conflicts that you haven't taken care off.

If you're not sure how to deal with merge conflicts then you should contact Rob

**Review by Endorze — CHANGES_REQUESTED** [review]
You need to solve the merge conflicts when they appear, if you're uncertain to how then try asking Rob for guidance.

**Review by Endorze — APPROVED** [review]
Looks great!

**Review by Meazajao — APPROVED** [review]

**Pull request merged**